doc/source/how-to/packaging.rst

```diff
@@ -334,23 +337,47 @@ The ``dependabot.yml`` file
 
 Dependabot version updates are performed by checking a ``dependabot.yml``
 configuration file into your repository. In this file, one should specify the
-location of the project's requirement files, so that Dependabot knows where
-to look.
+location of the project's requirement files, so that Dependabot knows where to
+look. The following code snippets show the required configuration for
+Dependabot according to the type of file in which the dependencies are
+specified:
```

Comment:
Should we talk as well about allowing dependabot to update your actions? I can't seem to find this info anywhere in the dev-guide, but I may be mistaken.

Reply:
I briefly cited the capabilities of dependabot for updating actions versions. On top of that, code snippets were updated with the required code for enabling such feature.